src/Maestro/SubscriptionActorService/PullRequestActor.cs

```diff
@@ -955,12 +954,16 @@ private async Task CalculatePRDescription(UpdateAssetsParameters update, List<De
                 coherencySection.AppendLine("attribute were produced in a build used as input to the parent dependency's build.");
                 coherencySection.AppendLine("See [Dependency Description Format](https://github.com/dotnet/arcade/blob/master/Documentation/DependencyDescriptionFormat.md#dependency-description-overview)");
                 coherencySection.AppendLine();
-
+                coherencySection.AppendLine("[DependencyUpdate]: <> (Begin)");
+                coherencySection.AppendLine();
+                coherencySection.AppendLine("**Coherency Update**:");
```

Comment:
I think this is redundant. 

The whole section already says that these are coherency updates:
![image](https://user-images.githubusercontent.com/23242101/87163399-26265100-c27c-11ea-984c-f2b198771696.png)

Reply:
Oh, am I correct in that you only added this string to facilitate the regex capture?

I think there are ways to prevent the need for this, like using two separate markers, one for dependency updates and one for coherency, and instead of capturing everything between the markers in a regex, iterate over both sections and extract the version updates.

If you want to keep the current aproach, this should follow the same pattern as the dependency updates:
 
- **Coherency updates:**
  - blah
  - blah

Reply:
If we are passing data that we, ourselves, should read, it shouldn't be text like this.  I'd suggest a JSON blob in a comment or something. <!-- MERGE INFORMATION DO NOT CHANGE : {"some json"} -->  Trying to make text that makes sense to humans AND is parsable by computers usually results in neither.

Reply:
@ChadNedzlek only problem with that approach is we have maximum limit for PR description as 4000 characters, by adding this in a json then we will run out of description space quickly.

Reply:
It seems like we probably shouldn't be trying to communicate to ourselves using the PR description as a transport layer.  We can do a bunch of human text parsing, but it's going to be fragile and error prone.  Is there a better place we can do this? Didn't we create this PR in the first place? Why can't we rebuild the information ourselves (or store it in some place less fragile)?

Reply:
Spoke to @ChadNedzlek we agreed on tackling the reconstruction of the PR description in another issue -> https://github.com/dotnet/core-eng/issues/10249